src/debugger/rnDebugSession.ts

```diff
@@ -189,7 +208,51 @@ export class RNDebugSession extends LoggingDebugSession {
         }
 
         super.disconnectRequest(response, args, request);
-   }
+    }
+
+    private establishDebugSession(resolve?: (value?: void | PromiseLike<void> | undefined) => void): void {
+        if (this.rnCdpProxy) {
+            const attachArguments = {
+                type: "pwa-node",
+                request: "attach",
+                name: "Attach",
+                continueOnAttach: true,
+                port: this.cdpProxyPort,
+                smartStep: false,
+                rnDebugSessionId: this.session.id,
```

Comment:
Maybe we need to add some explanation about this variable?

Reply:
Make sense. I added a description.